bigquery_etl/shredder/config.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+
+"""Meta data about tables and ids for self serve deletion."""
+
+from dataclasses import dataclass
+from functools import partial
+from itertools import chain
+from typing import Optional, Tuple
+
+
+SHARED_PROD = "moz-fx-data-shared-prod"
+
+
+@dataclass(frozen=True)
+class DeleteSource:
+    """Data class for deletion request source."""
+
+    table: str
+    id_: str
+    project: str = SHARED_PROD
+
+    @property
+    def sql_table_id(self):
+        """Fully qualified table id in standard sql format."""
+        return f"{self.project}.{self.table}"
+
+
+@dataclass(frozen=True)
+class DeleteTarget:
+    """Data class for deletion request target."""
+
+    table: str
+    id_: str
```

Comment:
Why is this suffixed with an underscore? Is the intention that it will never be read by the user and is thus considered private?